hydro_lang/src/runtime_support/resource_measurement.rs

```diff
@@ -1,28 +1,32 @@
+use std::cell::RefCell;
```

Comment:
I think this whole module will need to be `#[cfg(feature = "deploy_integration")]`? Cause Docker trybuild uses a different code path and won't need to pull in the `hydro_deploy_integration` dependency.

Reply:
`runtime_support` module already is featured gated on "runtime_support"

will double check at avoiding pulling in hydro_deploy_integration

Reply:
Oh I see